react-vite/generation-based-subscription/.env.example

```
@@ -0,0 +1,22 @@
+# Database Configuration
+# PostgreSQL connection string for Drizzle ORM
+# Example: postgresql://user:password@localhost:5432/dbname
+DATABASE_URL=postgresql://user:password@localhost:5432/dbname
+
+# Frontend base URL (optional - all instances default to localhost:5173)
+VITE_APP_URL="http://localhost:5173"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)